modules/standard/DateTime.chpl

```diff
@@ -186,10 +189,12 @@ module DateTime {
         month = 12;
         year -= 1;
       }
-      preceding -= daysInMonth(year, month);
+      const dim = try! daysInMonth(year, month);
+      preceding -= dim;
     }
     n -= preceding;
-    assert(0 <= n && n < daysInMonth(year, month));
+    const dim = try! daysInMonth(year, month);
+    assertInRange(n, 0, dim);
```

Comment:
This is off by one from the original assert `n < dim` vs. `n <= dim`.

Reply:
Shoot, good catch. Just force pushed a change that makes it `assertInRange(n+1, 1, dim);`